pages/update.js

```diff
@@ -38,6 +39,10 @@ class UpdatePage extends React.Component {
     return (
       <div className="UpdatePage">
         <Header collective={collective} LoggedInUser={LoggedInUser} />
+        <Head>
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <title>{`${updateSlug} - ${collective.name} - Open Collective`}</title>
+        </Head>
```

Comment:
- `updateSlug` is not a friendly string, you will get things like `annoucing-our-lastest-thing`, while what we want here is `Announcing our latest thing` - so the `update.title`. The `update` is not available in there because it's fetched from `UpdateWithData`. However, since we're only using this component in `pages/update`, we can safely refactor things to move the query up in the page (and eventually get rid of `UpdateWithData`).
- `Header` component supports setting a title through the `title` prop, so we can directly do:
```suggestion
        <Header collective={collective} LoggedInUser={LoggedInUser} title={this.getTitle()} />
```

Reply:
@Betree, So, we want to remove the `updateWithData` component completely, and bring that component's logic into the `pages/update`, Am I right, which means copy `updateWithData` component content and paste it in `pages/update.js` and adjust the necessary things like File imports, CComponent Structure etc... Am I right.

Reply:
Yes